Tests/Marketplace/upgrade_core_packs_list.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
use the same schema as `approved_usecasesd.json` (dict with comment and a list field) so it will be possible to expand the file in the future.

Reply:
Done